Packages/com.unity.inputsystem/InputSystem/Plugins/XInput/XboxGamepadMacOS.cs

```diff
@@ -111,7 +111,7 @@ internal struct XInputControllerWirelessOSXState : IInputStateTypeInfo
         public enum Button
         {
             Start = 11,
-            Select = 16,
+            Select = 10,
```

Comment:
Looks like a very targetted fix, just needs some QA to verify.

Reply:
I did wonder - Should we still expose the 'share' button too?
Not needed to fix the issue but seems a little odd we don't expose it.